frontend/src/modules/activity/components/integrations/devto/activity-devto-message.vue

```diff
@@ -19,8 +19,7 @@
     target="_blank"
   >
     {{
-      activity.articleTitle ||
-      activity.attributes.articleTitle
+      activity.channel || activity.attributes.articleTitle
```

Comment:
I believe this should be `activity.title` and not `activity.channel`
Also let's remove the console.log within `computedArgs`

Reply:
fixed to include title, even though channel included article title